community/graph-algo/src/main/java/org/neo4j/graphalgo/impl/centrality/EigenvectorCentralityArnoldi.java

```diff
@@ -95,6 +96,14 @@ public EigenvectorCentralityArnoldi( Direction relationDirection,
      */
     public Double getCentrality( Node node )
     {
+        if ( !nodeSet.contains( node ) )
+        {
+            return null;
+        }
+        if ( !node.hasRelationship() )
+        {
+            return 0d;
```

Comment:
exactly, it's not a flag it's a centrality of detached vertex. so would it be better to have DETACHED_VERTEX_CENTRALITY or smt similar?